src/shared/proto/robot_component_status.proto

```diff
@@ -0,0 +1,122 @@
+syntax = "proto3";
+
+/* Data about the status of the break beam */
+message BreakBeamStatus
+{
+    // Whether or not the ball is interrupting the robot’s laser beam.
+    bool ball_in_beam = 1;
+
+    // The reading of the robot’s laser sensor, normalized to [0,1]
+    float break_beam_reading = 2;
+}
+
+/* Data about the status of the chipper and kicker */
+message ChipperKickerStatus
+{
+    // Indicates the time elapsed since chipper was last fired in ms
+    // UINT64_MAX indicates chipper was never fired
+    uint64 time_since_chipper_fired = 1;
+
+    // Indicates the time elapsed since kicker was last fired in ms
+    // UINT64_MAX indicates kicker was never fired
+    uint64 time_since_kicker_fired = 2;
+}
+
+/* Data about the status of the dribbler */
+message DribblerStatus
+{
+    int32 dribbler_rpm = 1;
```

Comment:
Should this be unsigned? Assigning a sign to a direction here is fine, just pointing this out to be explicit. If we do want to assign a sign to a direction we should document it.

Reply:
added more comments

Reply:
Can we specify "along the shaft" or use the working we used for other motors in the new `app` interfaces? I can't seem to find the comments / exact wording so summoning @garethellis0

Reply:
I think this is what you're thinking of, it's from some IO code that we were waiting on hardware to test (so it's not merged yet):
```
/**
 * Set the rotation direction for the given motor driver
 *
 * Note that the rotation is from the perspective of rear of the motor, looking down
 * the shaft starting from the motor body
 *
 * @param motor_driver
 * @param direction
 */
```

Reply:
Yea this is what I was thinking of

Reply:
Added 
* Note that the rotation is from the perspective of rear of the motor, looking down
 * the shaft starting from the motor body
as a comment